src/libs/ReportUtils.js

```diff
@@ -82,11 +93,11 @@ function isReportMessageAttachment({text, html}) {
 /**
  * Given a collection of reports returns them sorted by last visited
  *
- * @param {Object} reports
+ * @param {Object} reportsToSort
```

Comment:
Instead of renaming all of these, we could just name the global reports variable to `allReports`

Reply:
Yeah, I considered both. If people like the global being `allReports` more, I'll change it back.